dandi/dandiarchive.py

```diff
@@ -442,7 +444,7 @@ class _dandi_url_parser:
                 flags=re.I,
             ),
             {"handle_redirect": "pass"},
-            "https://identifiers.org/DANDI:<dandiset id>[/<version id>] (<version id> cannot be 'draft')",
+            "https://identifiers.org/DANDI:<dandiset id>[/<version id>]",
```

Comment:
i would revert this change, since on identifiers.org version id indeed cannot be draft.

Reply:
indeed, I have `sed`ed too much... will rewrite and force-push